pylint/checkers/base/basic_checker.py

```diff
@@ -311,7 +311,7 @@ def _check_using_constant_test(
             nodes.Lambda,
             nodes.FunctionDef,
             nodes.ClassDef,
-            astroid.bases.Generator,
+            bases.Generator,
```

Comment:
I'm more reserved for bases / object / utils, is one supposed to know the internal organization of the astroid package by heart ?  nodes is easy to understand but those names are generic and not particularly evocative of anything tbh.